src/SmartComponents/modules/actions.js

```diff
@@ -70,7 +70,7 @@ function addStateFilter(filterData) {
 function filterByFact(filter) {
     return {
         type: types.FILTER_BY_FACT,
-        payload: filter
+        payload: filter.toLowerCase()
```

Comment:
I'd recommend to move the lowercasing to the reducer, if that's possible. The payload would stay original and the logic would be closer.